packages/ncids-js/src/components/nci-header/utils/focustrap.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * FOCUS TRAP
+ *
+ *
+ * Initialize the FocusTrap component:
+ * ```
+ * FocusTrap.element.create(HTMLElement);
+ * ```
+ */
+
+export class FocusTrap {
+	/** Active Parent Component. */
+	protected element: HTMLElement;
+	/** The Element that is the contains. */
+	protected context!: HTMLElement;
+	/** First focusable element */
+	protected firstFocusableElement!: HTMLElement;
+	/** Last focusable element */
+	protected lastFocusableElement!: HTMLElement;
+	/** List of element types to check */
+	protected focusableElements: string;
+	/** First focusable element */
+	protected focusableContent!: NodeListOf<Element>;
+	/** Callback for handling the toggle functionality.  */
+	private eventListener: EventListener = (event) => this.checkTrap(event);
+
+	/**
+	 * Sets component variables and initializes component.
+	 *
+	 * @param {HTMLElement} element Component being created.
+	 * @protected
+	 */
+	public constructor(element: HTMLElement) {
+		this.element = element;
+		this.focusableElements =
+			"button, [href], input, select, textarea, [tabindex]:not([tabindex='-1'])";
+	}
+
+	/**
+	 * Toggle the FocusTrap on or Off
+	 * @param {boolean} state State of the focus tgrap
+	 * @param {HTMLElement} context  the root container of the trap
+	 * @returns void
+	 */
+	public toggleTrap(state: boolean, context: HTMLElement): void {
+		if (state) {
+			this.findFocusableElements(context);
+			context.addEventListener('keydown', this.eventListener, true);
+		} else {
+			context.removeEventListener('keydown', this.eventListener, true);
+		}
+	}
+
+	/**
+	 *
+	 * @param {HTMLElement} element Find all focusable elements in the
+	 * context object
+	 * @returns void
+	 * @private
+	 */
+	private findFocusableElements(element: HTMLElement): void {
+		this.context = element;
+		this.focusableContent =
+			element.querySelectorAll(this.focusableElements) || [];
```

Comment:
Missing code coverage because of the `||`.

You can fix this by changing the property declaration to an array:
`protected focusableContent = [];`

and this line to 
```
		this.focusableContent = Array.from(
			element.querySelectorAll(this.focusableElements)
		);
```